ref(ourlogs): Clean frontend feature flags

k-fish opened the pull request:
### Summary
Cleans up the frontend FF for logs.

k-fish committed:
r#ef(ourlogs): Clean frontend feature flags

Cleans up the frontend FF for logs.

Zylphrex commented on static/app/views/explore/contexts/logs/logsPageData.tsx:
```diff
@@ -37,9 +38,6 @@ export function LogsPageDataProvider({children}: {children: React.ReactNode}) {
-  );
   const pageData = useLogsPageData();
-  return hasInfiniteFeature ? pageData.infiniteLogsQueryResult : pageData.logsQueryResult;
+  return pageData.infiniteLogsQueryResult;
```
Can we delete `logsQueryResult` from the context now?

k-fish replied:
Already was doing it

k-fish committed:
Get rid of old query too

k-fish committed:
Fix the circular import

k-fish committed:
Merge branch 'master' into ref/ourlogs/cleanup-feature-flags-fe

k-fish committed:
fix(build): Add missing asset loaders to chartcuterie build config

The recent frontend feature flag cleanup exposed new dependencies in the
chartcuterie bundle that require asset loaders for .svg, .png, .jpg, .gif,
.ico, .webp, and .pegjs files. Added these loaders to the esbuild config
to resolve build failures.

k-fish commented on config/build-chartcuterie.ts:
```diff
@@ -77,6 +77,21 @@ async function runEsbuild(commitHash: string): Promise<void> {
     treeShaking: true,
     logLevel: 'info',
+    loader: {
+      '.svg': 'file',
```
It looks as though we were getting lucky that the loaders existed before because of tree-shaking, we may not need these in charcuterie, but without them it'll fail to build

k-fish committed:
Merge remote-tracking branch 'origin/master' into ref/ourlogs/cleanup-feature-flags-fe
